cmd/main.go

```diff
@@ -176,15 +181,11 @@ func main() {
 
 	events := events.NewApi(eventsHandler, logrus.WithField("pkg", "eventsApi"))
 
-	if Options.DeployTarget == "k8s" {
-		expirer := imgexpirer.NewManager(log, s3Client.Client, Options.S3Config.S3Bucket, Options.BMConfig.ImageExpirationTime, eventsHandler)
-		imageExpirationMonitor := thread.New(
-			log.WithField("pkg", "image-expiration-monitor"), "Image Expiration Monitor", Options.ImageExpirationInterval, expirer.ExpirationTask)
-		imageExpirationMonitor.Start()
-		defer imageExpirationMonitor.Stop()
-	} else {
-		log.Info("Disabled image expiration monitor")
-	}
+	expirer := imgexpirer.NewManager(s3Client, eventsHandler, Options.BMConfig.ImageExpirationTime)
+	imageExpirationMonitor := thread.New(
```

Comment:
just a question why thread and not goroutine?

Reply:
It is actually a goroutine, look at pkg/thread in the repo.  It's just our wrapper that is useful for monitors.

Reply:
cool. interesting. didn't know we have such